recipe-server/client/control_new/state/recipes/selectors.js

```diff
@@ -90,3 +91,30 @@ export function getRecipeApprovalHistory(state, id) {
   const history = getRecipeHistory(state, id);
   return history.filter(revision => revision.get('approval_request'));
 }
+
+export function getRecipeFromURL(state) {
```

Comment:
we should be able to ditch this in favor of:
https://github.com/mozilla/normandy/pull/898/files#diff-cf8531aa9cc5d72ba402b1ed599a8dd7R23